app/Assets/Features.php

```diff
@@ -112,10 +112,8 @@ public static function when(string|array $featureNames, mixed $valIfTrue, mixed
 	{
 		// Sadly phpstan does not do the type inference as it would do in a if statement.
```

Comment:
Can this line go too?

Reply:
Done :)
[6f36776](https://github.com/LycheeOrg/Lychee/pull/2448/commits/6f367761f6e545c77c44d1623dc5c3c02b0bd0c2)